### PR TITLE
UX additions to groups SSO syncing

### PR DESCRIPTION
**Pull request by sellinjaanus**
When group syncing is disabled, now gray infromation box is displayed.
Also now in group members list, the person who added the person to group, is displayed.

### CODE REVIEW

File: frontend/src/component/admin/groups/GroupForm/GroupForm.tsx [hunk 1]
```diff
@@ -108,18 +137,48 @@ export const GroupForm: FC<IGroupForm> = ({
                     data-testid={UG_DESC_ID}
                 />
                 <ConditionallyRender
-                    condition={Boolean(uiConfig.flags.syncSSOGroups)}
+                    condition={Boolean(
+                        uiConfig.flags.syncSSOGroups && !isOss()
+                    )}
```
Comment (nunogois) [inline frontend/src/component/admin/groups/GroupForm/GroupForm.tsx:142]: Only need to wrap `uiConfig.flags.syncSSOGroups` in the `Boolean()`, not `isOss()`

File: frontend/src/component/admin/groups/GroupForm/GroupForm.tsx [hunk 2]
```diff
@@ -108,18 +137,48 @@ export const GroupForm: FC<IGroupForm> = ({
...
+                                    </StyledInputDescription>
+                                    <StyledItemList
+                                        label="SSO group ID / name"
+                                        value={mappingsSSO}
+                                        onChange={setMappingsSSO}
+                                    />
+                                </>
+                            }
+                            elseShow={() => (
+                                <>
```
Comment (nunogois) [inline frontend/src/component/admin/groups/GroupForm/GroupForm.tsx:160]: No need to wrap in a fragment, only 1 parent container: `StyledDescriptionBlock`.

**Review by nunogois — CHANGES_REQUESTED** [review]
LGTM - Noticed some little things we might want to change before merging.

File: frontend/src/component/admin/groups/GroupForm/GroupForm.tsx [hunk 3]
```diff
@@ -76,7 +95,17 @@ export const GroupForm: FC<IGroupForm> = ({
...
+
+    const { config: oidcSettings } = useAuthSettings('oidc');
+    const { config: samlSettings } = useAuthSettings('saml');
+
+    const isGroupSyncingEnabled = () => {
+        return (
+            (oidcSettings?.enabled && oidcSettings.enableGroupSyncing) ||
+            (samlSettings?.enabled && samlSettings.enableGroupSyncing)
+        );
+    };
```
[the remark this answers is not in the document]
Comment (sjaanus) [reply frontend/src/component/admin/groups/GroupForm/GroupForm.tsx:108]: Good idea! Done.

File: frontend/src/component/admin/groups/GroupForm/GroupForm.tsx [hunk 4]
```diff
@@ -108,18 +134,46 @@ export const GroupForm: FC<IGroupForm> = ({
...
+                                            title="You can enable SSO groups
+                                            syncronization if needed"
+                                            arrow
+                                        >
+                                            <StyledHelpOutline />
+                                        </Tooltip>
+                                    </div>
+                                    <Link data-loading to={`/admin/auth`}>
+                                        <span data-loading>
+                                            View SSO configuration{' '}
```
Comment (nunogois) [inline frontend/src/component/admin/groups/GroupForm/GroupForm.tsx:171]: Unneeded extra space?
```suggestion
                                            View SSO configuration
```

File: frontend/src/component/admin/groups/GroupForm/GroupForm.tsx [hunk 5]
```diff
@@ -108,18 +134,46 @@ export const GroupForm: FC<IGroupForm> = ({
...
-                                Is this group associated with SSO groups?
-                            </StyledInputDescription>
-                            <StyledItemList
-                                label="SSO group ID / name"
-                                value={mappingsSSO}
-                                onChange={setMappingsSSO}
-                            />
-                        </>
+                        <ConditionallyRender
+                            condition={Boolean(isGroupSyncingEnabled)}
```
Comment (nunogois) [inline frontend/src/component/admin/groups/GroupForm/GroupForm.tsx:142]: No need, it's already a boolean.
```suggestion
                            condition={isGroupSyncingEnabled}
```

**Review by nunogois — APPROVED** [review]
LGTM! 
I found some very small nitpicks in case you want to have a look, but I'm fine with this getting merged 👍

**Pull request merged**